oasis/llvm

```diff
@@ -18,7 +18,7 @@ Library llvm_plugin
                  Llvm_types,
                  Llvm_main
   CCOpt:         $cc_optimization
-  CCLib:         $llvm_lib $cxxlibs $llvm_ldflags
+  CCLib:         $llvm_lib $cxxlibs $llvm_ldflags -lpthread -lffi -ltinfo -ldl -lm
```

Comment:
this was a fast hack. I don't know why `llvm-config --ld-flags` stopped providing this options, maybe they moved it to `--libs` (that is more correct), maybe this is just a bug, and they forgot them. Can you try to find if there is any combination of options in `llvm-config-3.8` that will contain this five libraries?